code/modules/research/xenoarchaeology/artifact/artifact_losetta.dm

```diff
@@ -0,0 +1,42 @@
+/obj/structure/losetta_stone
+	name = "strange stone"
+	desc = "You feel a strange sense of emptiness, as if missing something, from looking upon this stone."
+	icon = 'icons/obj/xenoarchaeology.dmi'
+	icon_state = "loss"
+	var/chosen_language
+
+/obj/structure/losetta_stone/New()
+	..()
+	chosen_language = pick(LANGUAGE_HUMAN,\
```

Comment:
I believe you don't need those backslashes.

Also put the first entry on the next line, it looks nicer.